pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -627,12 +650,13 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication, ReentrancyGuard {
 
         uint256 executionDelay = _getDelayChangeExecutionDelay(_delaysPerActionId[actionId], newDelay);
 
-        bytes32 scheduleDelayActionId = getScheduleDelayActionId(actionId);
         bytes memory data = abi.encodeWithSelector(this.setDelay.selector, actionId, newDelay);
 
         // Since this can only be called by root, which is always a canceler for all scheduled executions, we don't
         // bother creating any new cancelers.
-        return _scheduleWithDelay(scheduleDelayActionId, address(this), data, executionDelay, executors);
+        uint256 scheduledExecutionId = _scheduleWithDelay(address(this), data, executionDelay, executors);
+        emit DelayChangeScheduled(actionId, scheduledExecutionId);
```

Comment:
I dropped `scheduleDelayActionId` in favor of `actionId`. I assume we don't need the specifier anymore because we now have specific event `DelayChangeScheduled`. But I might be wrong. Let me know.

Reply:
Correct!